mm2src/lp_ordermatch.rs

```diff
@@ -1547,54 +1656,281 @@ enum OrderbookRequestingState {
     NotRequested { subscribed_at: u64 },
 }
 
+type H64 = [u8; 8];
+
+struct TrieDiff<Key, Value> {
+    delta: Vec<(Key, Option<Value>)>,
+    next_root: H64,
+}
+
+impl<Key: std::fmt::Debug, Value: std::fmt::Debug> std::fmt::Debug for TrieDiff<Key, Value> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("TrieDiff")
+            .field("delta", &self.delta)
+            .field("next_root", &self.next_root)
+            .finish()
+    }
+}
+
+impl<Key: PartialEq, Value: PartialEq> PartialEq for TrieDiff<Key, Value> {
```

Comment:
The same comment

Reply:
The same response :slightly_smiling_face: